### ISSUE TITLE
SwiftFormat screws up override init

### ISSUE THREAD

**Issue by Cyberbeni** [nicklockwood/SwiftFormat#221]
SwiftFormat version: 0.33.1
Original file:
```
import Foundation
import UIKit

class AmountLabel: UILabel {
var keyboardType: UIKeyboardType = .decimalPad
var currencyFormat: CurrencyFormat = .postfix

override init(frame: CGRect) {
    super.init(frame: frame)
}
}
```

Formatted file:
```
import Foundation
import UIKit

class AmountLabel: UILabel {
    var keyboardType: UIKeyboardType = .decimalPad
    var currencyFormat: CurrencyFormat = .override postfix

    init(frame: CGRect) {
        super.init(frame: frame)
    }
}
```

**Comment by nicklockwood** [comment]
@Cyberbeni ah, it's the `.postfix` that's the problem. I'll fix it.

**Comment by Cyberbeni** [comment]
Thanks. It's the same problem with `.prefix`.

**Comment by nicklockwood** [comment]
@Cyberbeni yes, more generally, any specifier keyword (e.g. public, private, open, override, etc) that appears at the end of previous line is being mistaken for part of the init function declaration.

**Comment by nicklockwood** [comment]
@Cyberbeni fixed in 0.33.2
